blender/arm/material/make_voxel.py

```diff
@@ -11,19 +14,247 @@
 else:
     arm.enable_reload(__name__)
 
-
 def make(context_id):
     rpdat = arm.utils.get_rp()
-    con = make_ao(context_id)
+    if rpdat.rp_voxels == 'Voxel GI':
+        con = make_gi(context_id)
+    else:
+        con = make_ao(context_id)
 
     assets.vs_equal(con, assets.shader_cons['voxel_vert'])
     assets.fs_equal(con, assets.shader_cons['voxel_frag'])
     assets.gs_equal(con, assets.shader_cons['voxel_geom'])
 
     return con
 
+def make_gi(context_id):
+    con_voxel = mat_state.data.add_context({ 'name': context_id, 'depth_write': False, 'compare_mode': 'always', 'cull_mode': 'none', 'color_write_reds': [False], 'color_writes_green': [False], 'color_writes_blue': [False], 'color_write_alpha': False, 'conservative_raster': True })

```

Comment:
There's a small typo here: `color_write_reds` instead of `color_writes_red` I assume.